### PR TITLE
Story/quad 98

### PR DESCRIPTION
**Pull request by sarvani11**
@jdamerow Made changes as per comments. Please review and let us know if any changes are required. Contains changes of both stories (98 and 100).

### CODE REVIEW

**Comment by jdamerow** [comment]
ok to test

File: Quadriga/src/main/webapp/WEB-INF/views/top-navs/loggedin-nav.jsp [hunk 1]
```diff
@@ -6,6 +6,7 @@
 <nav id="nav" class="skel-ui-fixed">
 	<ul>
 		<sec:authorize access="hasAnyRole('ROLE_QUADRIGA_USER_ADMIN', 'ROLE_QUADRIGA_USER_STANDARD', 'ROLE_QUADRIGA_USER_RESTRICTED', 'ROLE_QUADRIGA_USER_COLLABORATOR')"><li ${currentPage == "home" ? "class=\"current_page_item\"" : ""}><a href="${pageContext.servletContext.contextPath}/auth/home">Home</a></li></sec:authorize>
+		<sec:authorize access="permitAll"><li ${currentPage == "home" ? "class=\"current_page_item\"" : ""}><a href="${pageContext.servletContext.contextPath}/sites">Sites</a></li></sec:authorize>
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/top-navs/loggedin-nav.jsp]: this tab is active whenever some is on the 'home' page:
image
Also, move that tab to the end after "Networks"

**Comment by jdamerow** [comment]
Looks great. Just this one small thing and we should be good to go.

**Comment by sarvani11** [comment]
@jdamerow Suggested changes are done and pushed. Please review and if any other changes please let us know.

File: Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml [hunk 2]
```diff
@@ -834,8 +834,22 @@
...
 	</definition>
-	
-	
+
+	<definition name="base.definition.public.sites" 
+		template="/WEB-INF/tiles/layouts/skeleton_public_sites.jsp" >
+		<put-attribute name="title" value="" />
+		<put-attribute name="navigation" value="" />
+		<put-attribute name="footer" value="/tiles/credits.jsp" />
+		<put-attribute name="currentPage" value="home" />
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/tiles-defs.xml]: you need to set this value to the one you use to make the tap active

**Comment by sarvani11** [comment]
@jdamerow: Modified as per comments

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/sites/SitesController.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	 * @return
+	 * @throws QuadrigaStorageException 
+	 */
+	@RequestMapping(value = "sites", method = RequestMethod.GET)
+	public String showQuadrigaPublicPages(Model model) throws QuadrigaStorageException {
+		List<IProject> projectList = retrieveProjectManager.getProjectListByAccessibility(EProjectAccessibility.PUBLIC.name());
+		model.addAttribute("projectList", projectList);
+		return "sites";
+	}
+}
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/sites/SitesController.java]: please read code style guide

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
@charan1312 and @MeghanTalasila, please correct formatting according to style guide, and then it should be ready to merge.

**Comment by jdamerow** [comment]
retest this please

**Pull request merged**